reports/manuscript/sections/conclusion.tex

```diff
@@ -1,27 +1,30 @@
 \section*{Conclusion}
 
-We present here a novel method for analysis of dMRI tractometry data. This
-method relies on the Sparse Group Lasso \cite{simon2013sparse} to provide both
-accurate prediction of the phenotypic properties of individual subjects based on
-their dMRI data, but also provides interpretable results by identifying the
-features that are important across subjects to make these predictions. The
-method is broadly applicable: it performs well in predicting both continuous
-variables, such as biological age, as well as discrete variable, such as whether
-a person is a patient or a healthy control. In both of these cases, SGL
-out-performs previous algorithms that have been developed for these tasks
-\cite{sarica2017corticospinal, Richard2018-ux}. The nested cross-validation
-approach used to fit the model and make both predictioons and inferences from it
-guards against overfitting and tunes the degree of sparseness required by the
-algorithm, such that both phenomena that are locally focused in a particular
-anatomical location and a particular property of diffusion (e.g., FA in the
-CST), as well as widely distributed phenomena can be accurately captured.
+We present here a novel method for analysis of dMRI tractometry data that relies
+on the Sparse Group Lasso \cite{simon2013sparse} to (a) make accurate
+predictions of phenotypic properties of individual subjects while,
+simultaneously, (b) identifying the most important features of the white matter
+in a completely data-driven approach. Importantly, through the use of a nested
+cross-validation procedure, the method adapts to the constraints of small
+datasets while also leveraging the power of larger datasets. Moreover, the
+method is broadly applicable to a wide range of research questions: it performs
+well in predicting both continuous variables, such as biological age, as well as
+categorical variables, such as whether a person is a patient or a healthy control.
+In both of these cases, SGL out-performs previous algorithms that have been
+developed for these tasks \cite{sarica2017corticospinal, Richard2018-ux}. The
+nested cross-validation approach used to fit the model and make both predictions
+and inferences from it guards against overfitting and tunes the degree of
+sparseness required by the algorithm. This means that both phenomena that are
+locally focused in a particular anatomical location and a particular property of
+diffusion (e.g., FA in the CST), as well as widely distributed phenomena can be
+accurately captured.
```

Comment:
Recommend changing this last sentence to active voice with itemization:
"Thus, SGL can accurately describe both (a) phenomena that are locally confined to a particular anatomical location or diffusion property (e.g., FA in the CST), as well as (b) phenomena that are widely distributed amongst brain regions and measured diffusion properties."

or something like that.

Reply:
Yes! Addressed in 53d9b86